src/interfaces/correction-interfaces/correct-name-interface.ts

```diff
@@ -19,6 +19,7 @@ export interface NrCorrectionIF {
 }
 
 export interface NrResponseIF {
```

Comment:
At some point we should commonize/share the NR IFs.

Create UI uses type "any" for the NR API response and then maps it to this:
https://github.com/bcgov/business-create-ui/blob/main/src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

... here:
https://github.com/bcgov/business-create-ui/blob/main/src/mixins/name-request-mixin.ts#L17

🤷‍♂️

Reply:
^^ not an action or request to change

Just something to keep in mind for sooner or later.

Reply:
Absolutely, i imagine this is quite a common request response!

Reply:
In Create UI this is NOT the request response. It's the local object.

Actually, we do have a shared IF for this:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/nameRequest-interface.ts

Ultimately we should have an IF (schema) for the API response -- the raw NR JSON.